manuscript/manuscript.Rmd

```diff
@@ -491,68 +491,51 @@ Many of our most realistic ecological models exist only as numerical simulations
+The flexibility of the approach makes it relatively straightforward to include realistic biology and environmental science in framing an RL problem, but training an RL agent to reach an acceptable level of performance can often be more challenging than expected.
 As the examples we have presented illustrate, it is important to begin with simple problems, including those for which an optimal strategy is already known.
 Such examples provide important benchmarks against which to calibrate the performance, tuning and training requirements of RL,
 and can easily be extended into more complex problems once RL agents can master the basics.
 Finding acceptable policies even in relatively simple problems may prove beyond the reach of existing algorithms and available computing power.
-<!-- Marcus is working on tightening up these next few paragraphs, precisely which issues actually get highlighted TBD-->
-In seeking either to adopt or assess a deep RL-based approach to an ecological issue, we highlight three key issues that are important to bear in mind: instability, interpretability, and reward design.
-
-
-<!-- reason 1: instability-->
-<!-- ML: I think this needs to be tightened up a bit-->
-One leading challenge for deep RL today is the problem of instability.
-RL agents trained on narrowly defined environments may perform poorly even when the environment is altered in seemingly trivial way, such as a re-scaling of parameters.
-To avoid this, researchers should seek to both train and evaluate agents against a range of environments representing different parameters or processes dynamics.
-Stable performance can be challenging even when using the same environment.
-For a range of reasons, including high variance [@cite] and the exploration vs. exploitation trade-off [@cite], model-free and model-based algorithms tend to have unstable performance on environments with non-tabular state-action spaces [cite RL that matters].
-For instance, it is not uncommon to observe vastly different behavior when an agent is initialized with different random seeds [cite ibid].
-Such unstable behavior can often be avoided by alternate selection of hyperparameters, so researchers should consider stability as well as performance during hyperparameter selection.
-The choice of RL algorithm, design of the reward function, and duration of training can also influence stability.
-
+In seeking either to adopt or assess a deep RL-based approach to an ecological issue, we highlight three key issues that are important to bear in mind: generalizability, interpretability, and reward design.
+
+<!-- reason 1: generalizability-->
+<!-- ML: pivoting this to generalizability but still rewrote past paragraph which might fit in the appendix somewhere-->
+Currently, deep RL agents struggle to generalize their skills [@pmlr-v97-cobbe19a].
```

Comment:
this is great but I think we need an example of what 'generalize their skills' means in an ecological context.  Or maybe just use language that is more concrete and less evocative of robots -- i.e. 'deep RL agents trained in one narrowly defined environment may struggle to perform well even on a closely related environment'?